#Define a function giving a NameError
def addition(x,y):
    z=x+y
    return z
    
print('This function gives a NameError')
total = add(3.14, 1.618)
print(total)